index.bs

```diff
@@ -305,6 +307,8 @@ or else the <a>top-level browsing context</a>'s {{document}} node
 (referred to as the <dfn for="IntersectionObserver">implicit root</dfn>) if
 the {{IntersectionObserver/root}} attribute is <code>null</code>.
 
+An {{Element}} is defined as having a <dfn for="IntersectionObserver">content clip</dfn> if its computed style has <a>overflow properties</a> that cause its content to be clipped to the element's padding edge.
```

Comment:
Hmm, this makes me think... this doesn't deal with the `clip` property at all, and it maybe should?